cmd/scaffold.go

```diff
@@ -1740,9 +1739,6 @@ func (fnb *FlowNodeBuilder) Build() (Node, error) {
 
 func (fnb *FlowNodeBuilder) onStart() error {
 
-	// seed random generator
-	rand.Seed(time.Now().UnixNano())
```

Comment:
The purpose of all my PRs is to remove the use of `math/rand` completely from production. There is no need to initialize `math/rand` anymore since it's not used by the node software.